test/metabase/cmd/rotate_encryption_key_test.clj

```diff
@@ -29,26 +27,6 @@
 
 (use-fixtures :once (fixtures/initialize :db))
 
-(defn- do-with-encrypted-json-caching-disabled
-  [thunk]
-  (let [mf (methodical/add-primary-method
-            @#'models/type-fn
-            [:encrypted-json :out]
-            (fn [_next-method _type _direction]
-              #'mi/encrypted-json-out))]
-    (with-redefs [models/type-fn mf]
-      (thunk))))
-
-(defmacro ^:private with-encrypted-json-caching-disabled
```

Comment:
this macro was meant to replace out transform for `encrypted-json` in toucan1, we now do that with the `with-redefs` for `transform-encrypted-json` on line 60, so this macro is no longer needed.